lms/djangoapps/organizations/views.py

```diff
@@ -100,7 +101,7 @@ def users(self, request, pk):
             * view parameter can be used to get a particular data .i.e. view=ids to
             * get list of user ids
         - POST: Adds a User to an Organization
-
+        - DELETE: Removes user link to an Organization for given users in `users` param
```

Comment:
This migt be more consistent with the descriptions of other methods:
DELETE: Removes the user(s) given in the `users` param from an Organization.